### PR TITLE
Include time to read article group and show ticks on read articles

### PR DESCRIPTION
**Pull request by richtier**
[this task](https://uktrade.atlassian.net/browse/ED-2476)
[this task](https://uktrade.atlassian.net/browse/ED-2477)
[this task](https://uktrade.atlassian.net/browse/ED-2417)

read-time

read-ticks

### CODE REVIEW

File: article/helpers.py [hunk 1]
```diff
@@ -39,15 +39,15 @@ def count_average_word_number_in_lines_list(lines_list, word_length=5):
...
+def time_to_read_in_seconds(article):
     """Return time to read in minutes give an Article object."""
     html = markdown_to_html(article.markdown_file_path)
     lines_list = lines_list_from_html(html)
     filtered_lines_list = filter_lines(lines_list)
     total_words_count = count_average_word_number_in_lines_list(
         filtered_lines_list
     )
-    return round(total_words_count / WORDS_PER_MINUTE, 2)
+    return round(total_words_count / WORDS_PER_SECOND, 0)
```
Comment (sdonk) [inline article/helpers.py]: The `ndigits` default is `0`, we can omit it.

You may want to change it [here](https://github.com/uktrade/directory-ui-export-readiness/blob/master/article/structure.py#L22) as well

Comment (konradko) [reply article/helpers.py]: Probably best to remove the rounding and leave it for the display layer

File: article/structure.py [hunk 2]
```diff
@@ -19,7 +19,7 @@ def __init__(self, articles, key, title, url):
     def total_reading_time(self):
         return round(
             sum((article.time_to_read for article in self.articles)),
-            2
```
Comment (konradko) [inline article/structure.py]: ditto

File: article/templatetags/article_tags.py [hunk 3]
```diff
@@ -11,3 +11,12 @@
 def include_markdown(markdown_file_path):
     html = helpers.markdown_to_html(markdown_file_path)
     return mark_safe(html)
+
+
+@register.filter
+def time_to_read(value):
+    if value <= 59:
+        return 'Less than 1 minute'
+    minutes = int(round(value / 60, 0))
```
Comment (konradko) [inline article/templatetags/article_tags.py]: might be slightly more readable if named `rounded_value`

**Pull request merged**